cypress/support/fragments/inventory/inventoryInstance.js

```diff
@@ -101,6 +120,43 @@ export default {
   goToHoldingView: () => {
     cy.do(viewHoldingsButton.click());
     HoldingsRecordView.waitLoading();
+  },
+
+  openHoldings: () => {
+    cy.do([
+      firstHolding.clickHeader(),
+      secondHolding.clickHeader()
+    ]);
+  },
+
+  verifySuccessCalloutMovingMessage(count) {
+    cy.expect(callout().is({ textContent: `${count} item has been successfully moved.` }));
+  },
+
+  moveItemToAnotherHolding() {
+    this.openHoldings();
+
+    cy.do([
+      firstHolding.find(MultiColumnListRow()).find(Checkbox()).click(),
+      firstHolding.find(Dropdown({ label: 'Move to' })).choose(including('Annex K1')),
```

Comment:
interactor can be moved in constant

Reply:
Interactor already in constant:

`const firstHolding = Accordion({ label: including('Holdings: Main Library') });`

In this case I use my constant for checking elements inside